src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,52 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int CHARACTERS_LIMIT = 6;
+    private static final int APPROPRIATE_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        nullChecking(user);
```

Comment:
I like how you divide your code with methods, but here are some little problems.
1. naming -- you have method with nullChecking naming and you pass user object. From naming, it means that you wanna check user for null. But you are checking fields as well.
2. you are mixing methods with if in your code.
Consider to choose methods or avoid them and check all in register method with if statements (then provide passwordChecking as well)